### ISSUE TITLE
newest version or 0.8.17

### ISSUE THREAD

**Issue by Jmattwilson** [FlodGamedev/OceanCraftBugtracker#52]
i cant load in a world it crashed my game

**Comment by theblackhawk333** [comment]
Same

**Comment by FlodGamedev** [comment]
on which platform are you playing?
did this happen in multiplayer or singleplayer. which region are you playing on.

did your headset run out of power during playing?

**Comment by FlodGamedev** [comment]
> Same

hey - could you please add the information i asked in the other reply, too!

**Comment by theblackhawk333** [comment]
I'm on quest 2 single player and my headset did not run out of battery.

**Comment by FlodGamedev** [comment]
hmm.. anything unusual - very large raft? lot of chests … any clue for us?

and i need to ask this. did you try rebooting your quest. and does the quest have storage space left?

**Comment by theblackhawk333** [comment]
Yes I rebooted my quest and I have 10 gigabytes of storage left. My raft
was 6x6 so fairly small and I had no chests

**Comment by Jmattwilson** [comment]
i tried to start a new world it crashed my game

**Comment by theblackhawk333** [comment]
> 

It seems like oceancraft is experiencing this issue for everybody if we give the devs time to investigate the issue they might be able so solve it.

**Comment by 4thHorsemanVR** [comment]
Game is crashing even after app data wipe, uninstall and reinstall.  On Quest 2 with plenty of free space (30+ GB).
Tried to play with several friends earlier and nobody could host a game.  Would kick us back to Quest home.
Status update please.
